java-support/src/main/java/io/cloudstate/javasupport/CloudState.java

```diff
@@ -300,6 +304,75 @@ public CloudState registerAction(
     return this;
   }
 
+  /**
+   * Register a annotated value entity.
+   *
+   * <p>The entity class must be annotated with {@link ValueEntity}.
+   *
+   * @param entityClass The entity class.
+   * @param descriptor The descriptor for the service that this entity implements.
+   * @param additionalDescriptors Any additional descriptors that should be used to look up protobuf
+   *     types when needed.
+   * @return This stateful service builder.
+   */
+  public CloudState registerValueEntity(
```

Comment:
On the user side, in the language supports, I think we can just use `entity` for these. So refer to them as `Entity` and use `registerEntity` and so on, while event sourced or replicated are specialisations.